gems/pending/spec/support/test_env_helper.rb

```diff
@@ -0,0 +1,177 @@
+require 'yaml'
+require 'fileutils'
+require 'pathname'
+require 'vcr'
+
+GEMS_PENDING_ROOT ||= File.expand_path(File.join(__dir__, "../.."))
+
+class TestEnvHelper
+  attr_reader :relative_dir, :absolute_recording_dir, :config_file
+
+  SPEC_DIR       = File.join(GEMS_PENDING_ROOT, 'spec')
+  SPEC_PATH      = Pathname.new(SPEC_DIR)
```

Comment:
I don't think you need both of these; Pathnames can be used almost completely interchangeably with Strings (File.join), and the one place you use `SPEC_DIR` can certainly just use a Pathname: `16: File.join(SPEC_PATH, 'test_env')`